R/format_PET.R

```diff
@@ -0,0 +1,1200 @@
+             Age = Age_observed,
+             Date = CaptureDate,
+             Year = BreedingSeason,
+             showpb = TRUE) %>%
+    #### Final arrangement
+    dplyr::select(.data$IndvID, .data$Species,
+                  .data$Sex_observed, .data$BreedingSeason,
+                  .data$CaptureDate, .data$CaptureTime,
+                  .data$ObserverID, .data$LocationID,
+                  .data$CaptureAlive, .data$ReleaseAlive,
+                  .data$CapturePopID, .data$CapturePlot,
+                  .data$ReleasePopID, .data$ReleasePlot,
+                  .data$Mass, .data$Tarsus, .data$OriginalTarsusMethod,
+                  .data$WingLength, .data$Age_observed, .data$Age_calculated,
+                  .data$ChickAge, .data$ExperimentID,
+                  #### mantain also BroodID needed to create Individual_data, remove at final step
+                  .data$BroodID)
+
+
+
+  Capture_data_jyntor <-
+    bind_rows(Capture_data_jyntor_adults,
+              Capture_data_jyntor_chicks) %>%
+    #### Create new variables
+    dplyr::group_by(.data$IndvID) %>%
+    dplyr::arrange(.data$BreedingSeason, .data$CaptureDate) %>% ## Use .data$CaptureDate if the data owner provides one. Now use StartDateOfLaying1May1.
+    dplyr::mutate(CaptureID = paste(.data$IndvID, row_number(), sep = "_")) %>%
+    dplyr::ungroup() %>%
+    #### Final arrangement
+    dplyr::select(.data$CaptureID, .data$IndvID, .data$Species,
+                  .data$Sex_observed, .data$BreedingSeason,
+                  .data$CaptureDate, .data$CaptureTime,
+                  .data$ObserverID, .data$LocationID,
+                  .data$CaptureAlive, .data$ReleaseAlive,
+                  .data$CapturePopID, .data$CapturePlot,
+                  .data$ReleasePopID, .data$ReleasePlot,
+                  .data$Mass, .data$Tarsus, .data$OriginalTarsusMethod,
+                  .data$WingLength, .data$Age_observed, .data$Age_calculated,
+                  .data$ChickAge, .data$ExperimentID,
+                  #### mantain also BroodID needed to create Individual_data, remove at final step
+                  .data$BroodID)
+
+  #### ------------------------
+  #### FINAL CAPTURE DATA
+  #### ------------------------
+  Capture_data <- bind_rows(Capture_data_parmaj,
+                            Capture_data_fichyp,
+                            Capture_data_jyntor)
+
+  return(Capture_data)
+}
+
+
+#### INDIVIDUAL DATA
+
+#' Create individual table for Institute of Biology, Russia.
+#'
+#' Create full individual data table in standard format for data from Institute of Biology, Russia.
+#' @param Capture_data Data frame, output of create_capture_PET function.
+#' @return A data frame.
+
+
+create_individual_PET <- function(Capture_data){
+
+  Individual_data <-
+    Capture_data %>%
+    #### Format and create new data columns
+    group_by(.data$IndvID) %>%
+    dplyr::summarise(Sex_calculated = purrr::map_chr(.x = list(unique(na.omit(.data$Sex_observed))),
```

Comment:
Make sure all non-base functions have explicit namespace. This flags as a NOTE in `devtools::check()`. See some discussion of this in the developer workflow https://github.com/SPI-Birds/pipelines#devtoolscheck 

```suggestion
    dplyr::summarise(Sex_calculated = purrr::map_chr(.x = list(unique(stats::na.omit(.data$Sex_observed))),
```